File: TestingSystem/API/Tests.Api/Tests.Api/Models/CreateQuestionModel.cs
namespace Tests.Api.Models;

public class CreateQuestionModel
{
    public string? QuestionName { get; set; }
    public string? ExpectedAnswer { get; set; }
    public string? QuestionType { get; set; }
    public IEnumerable<CreateSelectableAnswerModel>? SelectableAnswers { get; set; }
}

File: TestingSystem/Frontend/src/app/core/models/create-question-model.ts
import {CreateSelectableAnswerModel} from "./create-selectable-answer-model";

export class CreateQuestionModel {
  constructor(
    public questionName?: string,
    public expectedAnswer?: string,
    public questionType?: string,
    public selectableAnswers: CreateSelectableAnswerModel[] = []
  ) {
  }
}
